pkg/api/controllers/swagger.yaml

```diff
@@ -1247,16 +1247,16 @@ components:
       properties:
         page_size:
           type: integer
-          example: 1
+          example: 15
         has_more:
           type: boolean
           example: false
         previous:
           type: string
-          example: YXVsdCBhbmQgYSBtYXhpbXVtIG1heF9yZXN1bHRzLol=
+          example: "YXVsdCBhbmQgYSBtYXhpbXVtIG1heF9yZXN1bHRzLol="
```

Comment:
In yaml, this value already reads as a string, so the use of quotes aren't needed here.

Reply:
Hi Lucas! Thank you for your comment 😃
I know that the quotes here are not necessary. Because I had to change line 1259 to an empty string, I prefer the syntax to be homogeneous on both lines. What do you think?

Reply:
Hi Lucas! Thank you for your comment 😃
I know that the quotes here are not necessary. Because I had to change line 1259 to an empty string, I prefer the syntax to be homogeneous on both lines. What do you think?